src/client/app/composables/head.ts

```diff
@@ -61,3 +63,15 @@ function createHeadElement([tag, attrs, innerHTML]: HeadConfig) {
   }
   return el
 }
+
+function isMetaDescription(headConfig: HeadConfig) {
+  return (
+    headConfig[0] === 'meta' &&
+    headConfig[1] &&
+    headConfig[1].name === 'description'
+  )
+}
+
+function rejectHeadDescription(head: HeadConfig[]) {
```

Comment:
Maybe we can move these functions to `src/shared` so we don't duplicate the implementation across `node` and `client`? 

Also, thinking maybe `filterHeadDescription` could be more clear?

Reply:
Totally agree that needs to be moved to shared, but I tried to create a @shared/head file for these helpers before sending the PR but got compilation errors that I did not know how to fix.

About `filterHeadDescription`, at least for me, it sounds that you are going to get only the head description (instead of filtering it out). I associate it with the positive check condition like `filterApples === filter( _ , isApple )`. I used reject as in https://lodash.com/docs/4.17.15#reject. But not an issue if that is changed 👍🏼

Reply:
I tried again to move `rejectHeadDescription` to `shared/config.ts`, importing it as `../shared/config` from node and `/@shared/config` from client. This is the error I do not know how to resolve:
```
λ yarn docs-dev
yarn run v1.22.10
$ node ./bin/vitepress dev docs
vitepress v0.9.0
vite v1.0.0-rc.13
internal/modules/cjs/loader.js:1032
  throw err;
  ^

Error: Cannot find module '../dist/node'
Require stack:
- C:\Labs\vitepress\bin\vitepress.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:1029:15)
    at Function.Module._load (internal/modules/cjs/loader.js:898:27)
    at Module.require (internal/modules/cjs/loader.js:1089:19)
    at require (internal/modules/cjs/helpers.js:73:18)
    at Object.<anonymous> (C:\Labs\vitepress\bin\vitepress.js:16:3)
    at Module._compile (internal/modules/cjs/loader.js:1200:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1220:10)
    at Module.load (internal/modules/cjs/loader.js:1049:32)
    at Function.Module._load (internal/modules/cjs/loader.js:937:14)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:71:12) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [ 'C:\\Labs\\vitepress\\bin\\vitepress.js' ]
}
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

FWIW, I think the structure of shared code in Vitepress may change quite a bit once Vitepress is migrated to Vite 2 and this kind of helpers would end up in a rollup plugin shared by both dev and build processes

Reply:
Sorry for the late check in! Do you have any update on this one? @matias-capeletto 👀  Well Vite 2 here, and maybe we could tackle rollup plugin approach, though I think that's out of this PR's scope.

So maybe renaming method name and lets merge this...?

Reply:
Yes, I think if this is going to move to a plugin, the redundancy will be shortlived anyways.
I renamed `reject` to `filterOut`, I hope that is clearer. 
I also merged master into the branch and everything still looks good.